### ISSUE TITLE
Add tests for regexp

### ISSUE THREAD

**Issue by amznpurple** [awslabs/yesiscan#19]
We should add some test cases to make sure people are using the regexp functionality properly.

**Comment by ovalba** [comment]
I agree, this is important for Lite MVP mostly because we will depend heavily on this feature

**Comment by amznpurple** [comment]
I can write the test framework and @tapdiego-amzn can add some examples.

**Comment by ovalba** [comment]
Kicked out of MVP
